wondrous-app/utils/constants.ts

```diff
@@ -514,3 +514,26 @@ export const PROPOSAL_VOTE_LABELS = {
     ACTION: 'Reject',
   },
 };
+
+export const PAGES_WITH_NO_SIDEBAR = [
```

Comment:
nice

Reply:
gonna set up a wiki/notion doc that reminds to add to this list

Reply:
nice!